ipemu/csrc/spdlog-ext.h

```diff
@@ -221,11 +221,11 @@ class JsonLogger {
 };
 
 // Exported symbols
-#define FATAL(context)                                                         \
+#define FATAL(context) {                                                       \
   auto _fatal_fmt_msg = fmt::format("{}", context);                            \
   spdlog::critical("{}", context);                                             \
   spdlog::shutdown();                                                          \
-  throw std::runtime_error(_fatal_fmt_msg);
+  throw std::runtime_error(_fatal_fmt_msg); }
```

Comment:
required?

Reply:
No, just to make it simpler to use as a statement. E.g.

```
if(...) FATAL(...);
```